package.json

```diff
@@ -2,54 +2,59 @@
   "name": "ember-cli-opbeat",
   "version": "0.0.7",
   "description": "Add Opbeat exception logging to your Ember app.",
+  "keywords": [
+    "ember-addon"
+  ],
+  "license": "MIT",
+  "author": "",
```

Comment:
This key is now duplicated. (Nitpicking a single thing to prove that I read some of this.)